### ISSUE TITLE
Value of WGET_USER_AGENT not quoted correctly

### ISSUE THREAD

**Issue by sbrl** [pirate/ArchiveBox#129]
**Describe the bug**

I've noticed that the value of the `WGET_USER_AGENT` environment variable is not properly quoted by _ArchiveBox_. For example, I'm setting it to this in bash:

```bash
export WGET_USER_AGENT="ArchiveBox/$(git rev-parse HEAD | head -c7) (+https://github.com/pirate/ArchiveBox/) wget/$(wget --version | head -n1 | cut -d" " -f3)";
# Then call ArchiveBox as normal
./archivebox ....
```

This should result in something like `ArchiveBox/7890abc (+https://github.com/pirate/ArchiveBox/) wget/1.18`. The above causes a bogus _404 Not Found_ error on every request. However, changing it to this:

```bash
export WGET_USER_AGENT="\"ArchiveBox/$(git rev-parse HEAD | head -c7) (+https://github.com/pirate/ArchiveBox/) wget/$(wget --version | head -n1 | cut -d" " -f3)\"";
# Then call ArchiveBox as normal
./archivebox
```

....solves the issue.

**Expected behavior**

I shouldn't have to add the quotes manually myself in the `WGET_USER_AGENT` environment variable

**Screenshots**

Here it is not working:

![elessar_108](https://user-images.githubusercontent.com/9929737/51066576-ff457b00-1602-11e9-9575-295e72720b8c.png)

Here it is with the workaround:

![elessar_109](https://user-images.githubusercontent.com/9929737/51066595-13897800-1603-11e9-9a26-9d0eb0a0974d.png)

**Desktop (please complete the following information):**
 - OS: Raspbian GNU/Linux 9.6 (stretch) (headless)
 - Browser: (none)
 - Version [e.g. 22] ArchiveBox commit hash: 33942d37cb8b919215af03bf8aafe5443445a063

**Smartphone (please complete the following information):**
 - Device: n/a
 - OS: n/a
 - Browser n/a
 - Version [e.g. 22] n/a

**Additional context**

**Comment by pirate** [comment]
Thanks for the beautiful bug report! Fixed in b77476c, let me know if you're still having trouble and I'll re-open it.

**Comment by sbrl** [comment]
Yep - looks like that did the trick. Thanks! :D
